ts/src/pro/huobi.ts

```diff
@@ -87,7 +87,9 @@ export default class huobi extends huobiRest {
                 'tradesLimit': 1000,
                 'OHLCVLimit': 1000,
                 'api': 'api', // or api-aws for clients hosted on AWS
-                'maxOrderBookSyncAttempts': 3,
```

Comment:
@ttodua one thing is the orderbook fetching, and another is the orderbook synching, not sure if we should use a single option for both or not

Reply:
yes, that's I've mentioned too, so to ask - what should we name it? `snapshotSyncMaxRetries`? while we have `snapshotMaxRetries` for solely "fetching" retries.
or we should rename again. 
lmk your thoughts

Reply:
also, for existing `snapshotMaxRetries`, imo, can be renamed into `snapshotFetchMaxRetries`.

Reply:
@carlosmiei @ttodua I may be missing something, but if we're going to unify this option, then why not just call it `maxRetries` or something else as simple as that? When it comes to orderbook subscriptions there's just one process that "can be tried", that is synching the orderbook.

```
'options': {
    'watchOrderBook': {
        'maxRetries': 3,
    },
}
```

Reply:
@kroitor 
>  there's just one process that "can be tried", that is synching the orderbook.

Actually there are two, fetching the orderbook using rest (that can fail due to a temporary issue) and second align that orderbook with the deltas we cached

Reply:
Of course we can merge both options and make it one

Reply:
@carlosmiei i've merged, you can take a look now.